homework-3/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace LoggingLibrary
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            Logger.Instance.Log("Log message.");
+            Logger.Instance.Dispose();

```

Comment:
generally this is not called manually you can place the call of this function in destructor of the logger class